utils.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+)
+
+func (u UrlStoredObjectSet) toString() string {
+	raw := "["
+
+	for _, url := range u {
+		raw = raw + `{"url":"` + url.Url + `","tiny":"` + url.Tiny + `"},`
+	}
+
+	return raw
+}
+
+func ReadFromFIle(filename string) UrlStoredObjectSet {
+	hash, _ := ioutil.ReadFile(filename)
+	var urlStoredObject UrlStoredObjectSet
+
+	err := json.Unmarshal(hash, &urlStoredObject)
+
```

Comment:
remove blank lines ;) before err check 

also here you can do something like this

```go
if err := json.Unmarshal(hash, &urlStoredObject) {
return err
}
```